packages/inter-protocol/test/auction/test-auctionBook.js

```diff
@@ -14,8 +14,8 @@ import { makeOffer } from '@agoric/zoe/test/unitTests/makeOffer.js';
 import { setup } from '@agoric/zoe/test/unitTests/setupBasicMints.js';
```

Comment:
I'd like to see tests of adding and deleting (exiting) bids that show that vstorage is updated correctly.

Reply:
Already added. Check out `test/auction/snapshots/test-auctionContract.js.md`. The test *multiple bidders at one auction step* shows that `bid1001` has an empty node, while `bid1002` and `bid1003` show their data.